packages/clerk-js/src/core/clerk.ts

```diff
@@ -732,7 +732,10 @@ export class Clerk implements ClerkInterface {
     }
 
     // getToken syncs __session and __client_uat to cookies using events.TokenUpdate dispatched event.
-    await newSession?.getToken();
+    const token = await newSession?.getToken();
+    if (!token) {
+      eventBus.dispatch(events.TokenUpdate, { token: null });
+    }
```

Comment:
Don't we need to broadcast the sign out as well ?
```suggestion
    if (!token) {
      this.#broadcastSignOutEvent();
      eventBus.dispatch(events.TokenUpdate, { token: null });
    }
```

Reply:
This was a recent deletion we did so I brought it back, nice observation not 100% sure that we should broadcast the sign out as we mostly dispatching the `TokenUpdate` event to keep cookies clear if there is no token at that point

Reply:
I would expect to keep this without the `#broadcastSignOutEvent();` and probably add to this event a listener in clerk-js that call the `#broadcastSignOutEvent()` when it's the token is null.